style.css

```diff
@@ -82,6 +81,189 @@
   color: blue;
 }
 
+.header h1 {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+  margin-top: 50px;
+}
+
+.line hr {
+  display: none;
+  background: #36b37e;
+  width: 10%;
+  height: 5px;
+  border-radius: 24px;
+  margin-left: 45%;
+  margin-top: 40px;
+}
+
+.grid {
+  display: grid;
+  grid-template-columns: 100%;
+  grid-template-rows: 60% 40%;
+  grid-template-areas:
+    'grid1'
+    'grid2';
+  margin: 30px;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+  background: #ebf0ee;
+}
```

Comment:
- [ ] KIndly make sure these sections are flexboxes as per the project requirement, the only grid will be the container of all cards which I think is `,page-11 in your case.
- [ ] I also suggest fixing the broken design when we are still between 375 -768 px as it is our mobile version width interval.
![Screenshot from 2023-02-15 12-34-11](https://user-images.githubusercontent.com/97834160/219005996-57e50232-fcae-48cd-a188-07a47e1d163a.png)